# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../mapnik/autogen.patch
# Copyright (C) 2009 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./autogen.sh.orig	2009-08-18 02:22:58.000000000 +0300
+++ ./autogen.sh	2009-08-18 02:23:05.000000000 +0300
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-./bootstrap && ./configure $@
+./bootstrap
